mlflow/protos/service.proto

```diff
@@ -581,6 +583,8 @@ message CreateRun {
   optional string experiment_id = 1;
 
   // ID of the user executing the run.
+  // This field is deprecated, and will be removed in a future MLflow release.
```

Comment:
jfyi - we are planning on adding a doc page specific to MLflow tags, as opposed to documenting within the REST API docs. I think someone may have accidentally deleted your previous docs, but those will be resurrected there.